cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlMain.java

```diff
@@ -35,7 +35,7 @@ public static void main(String[] args) throws Exception {
     KafkaCruiseControlConfig config = readConfig(args[0]);
     Integer port = parsePort(args, config);
     String hostname = parseHostname(args, config);
-    KafkaCruiseControlApp app = new KafkaCruiseControlApp(config, port, hostname);
+    KafkaCruiseControlApp app = new KafkaCruiseControlVertxApp(config, port, hostname);
```

Comment:
We should keep backward compatibility, so please keep the servlet app here and add an argumentum to specify vertx mode.